docs/root/operations/admin.rst

```diff
@@ -490,9 +490,12 @@ modify different aspects of the server:
   Outputs /stats in `Prometheus <https://prometheus.io/docs/instrumenting/exposition_formats/>`_
   v0.0.4 format. This can be used to integrate with a Prometheus server.
 
-  You can optionally pass the ``usedonly`` URL query argument to only get statistics that
-  Envoy has updated (counters incremented at least once, gauges changed at least once,
-  and histograms added to at least once)
+  If the ``usedonly`` URL query argument is sent, Envoy will only return
+  statistics that it has updated (counters incremented at least once,
+  gauges changed at least once, and histograms added to at least once)
+
+  If the ``format=REGEX`` URL query argument is sent, Envoy will only return
+  statistics whose name matches the given regular expression.
```

Comment:
hi @jpeach im trying to understand this - does this mean that if i query with `/stats/prometheus?format=^foo` it will return stats with name beginning `foo` or is the regex passed through a different arg ?

Reply:
either way - i think we should add a trivial example explaining

Reply:
> hi @jpeach im trying to understand this - does this mean that if i query with `/stats/prometheus?format=^foo` it will return stats with name beginning `foo`

Yup, that's exactly right

Reply:
@jpeach if you can add a trivial example - even the one i posted above - into the text here, im happy to land this